regression/cbmc/CMakeLists.txt

```diff
@@ -22,16 +22,20 @@ add_test_pl_profile(
     ${gcc_only}
 )
 
-add_test_pl_profile(
+if("${CMAKE_SYSTEM_NAME}" STREQUAL "Windows")
+  message(WARNING "skipping broken cprover-smt2 tests on Windows")
```

Comment:
This is confusing, the `-X broken-smt-backend` already excludes the "broken" cprover-smt2 tests. Why do we need to exclude the rest of them as well. Short explanation will do.